src/Icons/src/DependencyInjection/UXIconsExtension.php

```diff
@@ -58,6 +58,10 @@ public function getConfigTreeBuilder(): TreeBuilder
                         ->end()
                     ->end()
                 ->end()
+                ->booleanNode('ignore_not_found')
+                    ->info('Ignore error when an icon is not found.')
+                    ->defaultFalse()
```

Comment:
It could default to `%kernel.debug%` here or in the recipe

Reply:
I think the idea in #2008 was to default to "not kernel.debug". 

I this change anything in prod, in would be a BC break, so maybe the recipe is a good solution, indeed